wandb/sdk/interface/interface.py

```diff
@@ -628,6 +629,7 @@ def publish_partial_history(
     ) -> None:
         run = run or self._run
 
+        data = serialize_custom_charts(run, data)
```

Comment:
also on second thought, maybe having it in the Run class makes more sense
and probably we should move the history_dict_to_json to the run as well...

Reply:
so i'll just rename the function and that it